fish_prompt.fish

```diff
@@ -151,7 +151,10 @@ function fish_prompt
     set prompt $prompt $pure_color_gray(basename "$VIRTUAL_ENV")"$pure_color_normal "
   end
 
-  set prompt $prompt "$color_symbol$pure_symbol_prompt$pure_color_normal "
+  # vi-mode indicator
+  set mode_indicator (fish_default_mode_prompt)
```

Comment:
Do we have a default value for `fish_default_mode_prompt`? If not specified, would this break the prompt?

Reply:
I believe it is part of fish-shell: https://github.com/fish-shell/fish-shell/blob/master/share/functions/fish_default_mode_prompt.fish